index.html

```diff
@@ -351,6 +351,142 @@ <h4 class="skills-text">Skills</h4>
           </li>
         </ul>
       </article>
+
+      <section class="contact-me">
+        <h2 class="primary-header content">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get
+          in touch.
+        </h2>
+
+        <form action="https://formspree.io/f/mdobkdnq" method="POST" class="forms margin-centre">
+          <input
+            class="forms1"
+            type="text"
+            id="name"
+            name="name"
+            placeholder="Full name"
+            required
+            maxlength="30"
+          />
+          <input
+            class="forms1"
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Email Address"
+            required
+          />
```

Comment:
I am getting an error if I were to use an email with `a@b` (see screenshot), can you [validate](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#validation) this email with a regex, see [pattern](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern)? 

![image](https://user-images.githubusercontent.com/36057474/147852494-4f44de90-a042-4a67-a12b-0f472d2a3932.png)